README.md

```diff
@@ -116,6 +116,8 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 
    ***NOTE*** The private key used to connect to the master node is by default (currently hardcoded) at ~/.ssh/id_ibmcloud, so make sure you copy the generated key's private key info into this file.
 
+   ***NOTE*** Make sure the pod CIDR and serivce CIDR you specified in `cluster.yaml` have no overlap with provisioned node CIDR, and each data center you specified in `machines.yaml` has different node CIDR setting, for complete node CIDR settings in all data centers, please refer to: https://control.softlayer.com/network/subnets
```

Comment:
How about move this to troubleshooting section by adding more detail for what is the problem if service cidr is same as vm cidr?

Reply:
Good advice.

Reply:
@gyliu513 Done